LICENSE

```diff
@@ -2,7 +2,7 @@ Cosmos-SDK
 License: Apache2.0
 
                                  Apache License
-                           Version 2.0, January 2004
+                           Version 2.0, January 2020
```

Comment:
This year should not change, since the year is based on the official approval date by the ASF.

Reply:
cool